checker.cpp

```diff
@@ -1,17 +1,63 @@
 #include <assert.h>
+#include<bits/stdc++.h>
+using namespace std;
 
+
+
+		 void Check_BPM (float bpm)
+		{
```

Comment:
This is very similar to the other Check... functions. So if the cout has to be replaced with another mechanism, it will result in multiple changes.
Try making one 'VitalsRangeChecker' which accepts the limits also as input. Then isolate the reporter (cout).